device-catalog.html

```diff
@@ -9,186 +9,224 @@
+                            <img src=".//img/item-1.jpg" alt="Любительская селфи-палка" max-width="360" max-height="380">
+                            <span>
+                                <a href="#"><h3>Любительская селфи-палка</h3></a>
+                                <p>500 руб.</p>
+                            </span>                            
+                            <div class="content-buttons">
+                                <button type="button">в корзину</button>
+                                <button type="button">Добавить к сравнению</button>
+                            </div>                        
+                        </li>
+                        <li class="content-item">
+                            <img src=".//img/item-2.jpg" alt="Профессиональная селфи-палка" max-width="360" max-height="380">
+                            <span>
+                                <a href="#"><h3>Профессиональная селфи-палка</h3></a>
+                                <p>1500 руб.</p>
+                            </span>                            
+                            <div class="content-buttons">
+                                <button type="button">в корзину</button>
+                                <button type="button">Добавить к сравнению</button>
+                            </div>
+                        </li>
+                        <li class="content-item">
+                            <img src=".//img/item-3.jpg" alt="Непотопляемая селфи-палка" max-width="360" max-height="380">
+                            <span>
+                                <a href="#"><h3>Непотопляемая селфи-палка</h3></a>
+                                <p>2500 руб.</p>
+                            </span>
+                            <div class="content-buttons">
+                                <button type="button">в корзину</button>
+                                <button type="button">Добавить к сравнению</button>
+                            </div>                        
+                        </li>
+                        <li class="content-item">
+                            <img src=".//img/item-4.jpg" alt="Селфи-палка «Следуй за мной»" max-width="360" max-height="380">
+                            <span>
+                                <a href="#"><h3>Селфи-палка «Следуй за мной»</h3></a>
+                                <p>4900 руб.</p>
+                            </span>
+                            <div class="content-buttons">
+                                <button type="button">в корзину</button>
+                                <button type="button">Добавить к сравнению</button>
+                            </div>                        
+                        </li>
+                    </ul>
+    
+                </div>
+                <div class="content-pages">
+                    <a class="page-back" href="#">назад</a>
+                    <ul>
+                        <li>1</li>
```

Comment:
in reality it is also a link, disabled when this is a current page

Reply:
Yep, I thought about it. Link